apps/keyboard/build/layout_configurator.js

```diff
@@ -151,6 +151,10 @@ KeyboardLayoutDetail.prototype.load = function(appDir) {
 
       break;
 
+    case 'thai':
```

Comment:
You should put this at line 137 before `case 'vietnamese':` instead of of creating another empty `case ... break`.